File: csharp/6-kyu/create-phone-number/solution.cs
public class Kata
{
  public static string CreatePhoneNumber(int[] numbers)
  {
    var phoneNumber = "(";
    
    for (var i = 0; i <= 2; i++)
      phoneNumber += numbers[i];
      
    phoneNumber += ") ";
    
    for (var i = 3; i <= 5; i++)
      phoneNumber += numbers[i];
      
    phoneNumber += "-";
    
    for (var i = 6; i <= 9; i++)
      phoneNumber += numbers[i];
      
    return phoneNumber;
    
  }
}

File: javascript/6-kyu/create-phone-number/solution.js
function createPhoneNumber(numbers) {
  let phoneNumber = "";

  numbers.map((num, index) => {
    if (index === 0) {
      phoneNumber += "(";
    } else if (index === 3) {
      phoneNumber += ") ";
    } else if (index === 6) {
      phoneNumber += "-";
    }

    phoneNumber += num.toString();
  });

  return phoneNumber;
}
